website/src/css/custom.less

```diff
@@ -63,15 +63,15 @@
   }
 
   .troubleshooting-report-area {
-    > span {
```

Comment:
@TucksonDev can you remind me how to locally run the linter/fixer that you recently configured? That should revert these formatting regressions, right?

Reply:
`yarn format` should run all formatters. Although I think .less files are not included. You can add it to the package.json file, in the `format:app` script, at the end => `\"src/**/*.{tsx,ts,scss,less,json,js}\"` (notice the `less` I added between `scss` and `json`)
Otherwise, as this is just a formatter (the linter is not configured yet), we can also deal with that in another PR.

Reply:
Will fiddle in a separate PR :)